tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "target": "es2015",
+    "moduleResolution": "node",
+    "noEmitOnError": true,
```

Comment:
Didn't notice this before. What does `noEmitOnError` mean?

Reply:
Generally a fail-safe to ensure it doesn't just warn then run, but fails completely. Its probably not needed as it defiantly was failing the test before without it.

Reply:
OK so it's just a confusing name :)